web/src/narrow_banner.ts

```diff
@@ -345,12 +345,24 @@ export function pick_empty_narrow_banner(): NarrowBannerData {
                         }),
                     };
                 }
+                const recipient = people.get_by_user_id(user_ids[0]);
```

Comment:
`recipient_user` would be a better variable name, and you could maybe do a prep commit refactoring to define that before the  `people.is_current_user(first_operand)`, check, and maybe have that check use `recipient_user` rather than `first_operand` for consistency.